PerfettoCds/Pipeline/CompositeDataCookers/PerfettoCpuUsageEventCooker.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Extensibility.DataCooking;
+using Microsoft.Performance.SDK.Processing;
+using PerfettoCds.Pipeline.DataOutput;
+using PerfettoProcessor;
+
+namespace PerfettoCds.Pipeline.DataCookers
+{
+    /// <summary>
+    /// Pulls data from multiple individual SQL tables and joins them to create a CPU usage event.
+    /// CPU usage events include multiple CPU counters polled throughout the trace from  the /proc/stat file.
+    /// Percent use values are calculated by comparing the difference in the counter between 2 adjacent events:
+    /// time2.counterAPercent = (time2.counterA - time1.counterA) / (time2 - time1) * 100
+    /// </summary>
+    public sealed class PerfettoCpuUsageEventCooker : CookedDataReflector, ICompositeDataCookerDescriptor
```

Comment:
Rename to something that includes CPUCounter ProcStat? To help disambiguate from at least the more traditional WPA CPU % based on CPU Scheduling or CPU sampling

Reply:
I'll change it to CpuCounters. I think adding ProcStat will be confusing